### DISCUSSION TITLE
Dont make everything in users collection editable

### DISCUSSION THREAD

**Discussion by HexiDev** [pocketbase/pocketbase#2631] [Q&A] [answered]
So I was looking at the documentation of https://pocketbase.io/docs/api-rules-and-filters#filters-syntax but could not find a way to make one or more fields not editable without disabling it all together

**Comment by ganigeorgiev** [comment, marked as answer]
You can use the `@request.data.*` and the [`:isset` filter modifier](https://pocketbase.io/docs/api-rules-and-filters/#isset-modifier) to prevent storing submitted values, something like:

```js
@request.data.someField:isset = false
```

**Reply by HexiDev** [reply]
Thanks! I will try it right away
